docs/audit_report/changes/topics/hss_lms.yml

```diff
@@ -1,5 +1,8 @@
 title: HSS/LMS
 
+description: |
```

Comment:
Somehow Philippe is missing as Code Author here in the PDF

Reply:
Currently the scripts don't take the `Co-Authored-By: ...` in the commits into account. Only the pull-request author. I'm guessing that's the reason it didn't pick up @lieser.

We could certainly look into this, though. Sucks that we're currently missing due credit. Perhaps GitHub even provides an API to resolve the email addresses in `Co-Authored-By` into a GitHub account? That'd certainly help.